api/time_bucket.md

```diff
@@ -102,12 +102,23 @@ GROUP BY month_bucket
 ORDER BY month_bucket DESC LIMIT 10;
 ```
 
+It is possible to compute buckets on version 7 UUIDs that contain a timestamp
+in Unix time format. When bucketing UUIDs, a regular `TIMESTAMP WITH TIME ZONE`
+value is returned instead of a UUID:
+
+```sql
+ SELECT time_bucket('1 hour', '0194214e-cd00-7000-a9a7-63f1416dab45'::uuid);
+      time_bucket
+------------------------
+ 2025-01-01 11:00:00+01
+```
+
```

Comment:
When you bucketing UUIDs, a regular `TIMESTAMP WITH TIME ZONE` value is returned.